docs/releases/1.8.txt

```diff
@@ -353,6 +353,10 @@ Forms
 * You can now pass a callable that returns an iterable of choices when
   instantiating a :class:`~django.forms.ChoiceField`.
 
+* :class:`~django.forms.CharField` now accepts a :attr:`strip` argument to
```

Comment:
To fix the build error, you need to reference the entire path:  
```
:attr:`~django.forms.CharField.strip`
```
and yes, this is too late for 1.8 as alpha marked the feature freeze.